packages/cactus-plugin-ledger-connector-fabric/src/main/typescript/get-prometheus-exporter-metrics/get-prometheus-exporter-metrics-endpoint-v1.ts

```diff
@@ -0,0 +1,80 @@
+import { Express, Request, Response } from "express";
+
+import {
+  Logger,
+  LoggerProvider,
+  LogLevelDesc,
+  Checks,
+} from "@hyperledger/cactus-common";
+
+import {
+  IWebServiceEndpoint,
+  IExpressRequestHandler,
+} from "@hyperledger/cactus-core-api";
+
+import OAS from "../../json/openapi.json";
+
+import { registerWebServiceEndpoint } from "@hyperledger/cactus-core";
+
+import { PluginLedgerConnectorFabric } from "../plugin-ledger-connector-fabric";
+
+export interface IGetPrometheusExporterMetricsEndpointV1Options {
+  logLevel?: LogLevelDesc;
+  connector: PluginLedgerConnectorFabric;
+}
+
+export class GetPrometheusExporterMetricsEndpointV1
+  implements IWebServiceEndpoint {
+  private readonly log: Logger;
+
+  constructor(
+    public readonly opts: IGetPrometheusExporterMetricsEndpointV1Options,
+  ) {
+    const fnTag = "GetPrometheusExporterMetricsEndpointV1#constructor()";
+
+    Checks.truthy(opts, `${fnTag} options`);
+    Checks.truthy(opts.connector, `${fnTag} options.connector`);
+
+    this.log = LoggerProvider.getOrCreate({
+      label: "get-prometheus-exporter-metrics-v1",
+      level: opts.logLevel || "INFO",
+    });
+  }
+
+  public getExpressRequestHandler(): IExpressRequestHandler {
+    return this.handleRequest.bind(this);
+  }
+
+  public getPath(): string {
+    return OAS.paths[
+      "/api/v1/plugins/@hyperledger/cactus-plugin-ledger-connector-fabric/get-prometheus-exporter-metrics"
+    ].get["x-hyperledger-cactus"].http.path;
+  }
+
+  public getVerbLowerCase(): string {
+    return OAS.paths[
+      "/api/v1/plugins/@hyperledger/cactus-plugin-ledger-connector-fabric/get-prometheus-exporter-metrics"
+    ].get["x-hyperledger-cactus"].http.verbLowerCase;
+  }
+
+  public registerExpress(app: Express): IWebServiceEndpoint {
+    registerWebServiceEndpoint(app, this);
+    return this;
+  }
+
+  async handleRequest(req: Request, res: Response): Promise<void> {
+    const fnTag = "GetPrometheusExporterMetrics#handleRequest()";
+    this.log.debug(`POST ${this.getPath()}`);
+
+    try {
+      const resBody = await this.opts.connector.getPrometheusExporterMetrics();
+      res.status(200);
+      res.send(resBody.result.replace(/\\/g, ""));
```

Comment:
@jagpreetsinghsasan If you are returning plain text (which I'm guessing Prometheus forces you to do) then the response definition in openapi.json should match that as well. Right now it looks like there's a disconnect between what the openapi.json specifies and what the endpoint implementation actually returns

Reply:
The issue is, the response definition looks like
![image](https://user-images.githubusercontent.com/56861721/108477595-6d1c0080-72b9-11eb-8391-21d2ea1e9cda.png)

But the response I am getting from the server is something like
'# HELP cactus_fabric_total_tx_count Total transactions executed\n# TYPE cactus_fabric_total_tx_count gauge\n     'cactus_fabric_total_tx_count{type=\"cactus_fabric_total_tx_count\"} 3';

It is escaping the double quotes using '\'
Which is not acceptable by prometheus
So I used the above code to remove the same. (using replace fx)
Is there any other way to fix it?

Reply:
@jagpreetsinghsasan I've removed the replacement logic and it was working fine in both the automated tests and the manual tests (where I actually pull up prometheus and see if the graph gets charted properly with the total tx count).
Here are my changes on top of your branch that I'd recommend applying as part of this change request: https://github.com/petermetz/cactus/commit/82aa01cf431ef1e2e947f6d6470151508e6f861b

Reply:
Fixed.